app/forms/admin/depositors_search_form.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Admin
+  # Admin form object for searching for depositor information
+  class DepositorsSearchForm < ApplicationForm
+    before_validation :normalize_druids
+    attribute :druids, :string
+    validate :works_present
```

Comment:
It feels strange to me to invalidate the form if a druid isn't found, wouldn't that invalidate the whole process? Perhaps a way to list druids that aren't found so that individual druids in a long list do not invalidate the entire form. I could be misunderstanding what's happening here though.

Reply:
It's perhaps (❓) surprising how this set of deltas in this branch leads to the following behavior which I *think* may be what we want:

![Peek 2025-03-03 16-45](https://github.com/user-attachments/assets/7e3555d4-8d10-4761-aaad-a79aa7aac4e7)